ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestQueryTranslationTest.java

```diff
@@ -87,6 +87,9 @@
           KsqlConfig.KSQL_STREAMS_PREFIX + StreamsConfig.PROCESSING_GUARANTEE_CONFIG,
           StreamsConfig.EXACTLY_ONCE_V2 // To stabilize tests
       )
+      // Setting to anything lower will cause the tests to fail because we won't correctly commit
+      // transaction marker offsets.
+      .withProperty(KsqlConfig.KSQL_STREAMS_PREFIX + StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 2000)
```

Comment:
Thanks. You were concerned that this might introduce flakiness, and I think you're right. But we can also cross that bridge later.

Namely, if the actual results are sensitive to the relative timing of processing and committing, then when the test agent is slower or faster, we may get different results.